### PR TITLE
Tidy up profiles and add help

### PR DESCRIPTION
**Pull request by kris-nova**
Just cleaning up our profile work,
adding profiles to help and refactoring file/function names.

### CODE REVIEW

File: cmd/create.go [hunk 1]
```diff
@@ -148,3 +181,48 @@ func expandPath(path string) string {
 	}
 	return path
 }
+
+var (
+	p = func() string {
+		str := ""
+		spaces := ""
+		maxLen := 0
+		for shorthand := range profileMapIndexed {
```
Comment (kris-nova) [inline cmd/create.go]: I know it's `O(2n)` but I think it's necessary.. meep

**Comment by kris-nova** [comment]
Can another maintainer take a look and merge this please?

File: cmd/create.go [hunk 2]
```diff
@@ -71,27 +72,59 @@ func init() {
...
-	"do":           profiles.NewSimpleDigitalOceanCluster,
-	"digitalocean": profiles.NewSimpleDigitalOceanCluster,
-	"do-centos":    profiles.CentosDigitalOceanCluster,
-	"aws-centos":   profiles.CentosAmazonCluster,
+type profileMap struct {
+	profileFunc profileFunc
+	description string
+}
+
+var profileMapIndexed = map[string]profileMap{
```
Comment (prateekgogia) [inline cmd/create.go]: I fear this list can be endless, centOS on GCE, DO, AWS, Azure, Packet, IBM .. Ubuntu on GCE, DO, AWS, Packet, IBM. We should have some generic logic for this stuff, which is common and distributions / CP specific logic can keep growing.

Comment (kris-nova) [inline cmd/create.go]: I agree - I think we need a map that maps items from 2 lists together. Can we follow up PR for now and get this in as is?

Comment (prateekgogia) [inline cmd/create.go]: Yes, I can. Can you please fix the conflicts.

**Pull request merged**